netkan/netkan/spacedock_adder.py

```diff
@@ -88,6 +88,13 @@ def try_add(self, info):
         )
         return True
 
+    @staticmethod
+    def deletion_msg(msg):
```

Comment:
Why are we setting the staticmethod decorator here? Do we expect to use it outside of the class without instantiation?

Reply:
Ahh, I see it now.